client/5001-ephemeral-consumer-behavior

```diff
@@ -0,0 +1,18 @@
+# 5001-ephemeral-consumer-behavior
```

Comment:
I suspect not keeping them by sequential number will break the cli tools that exist to make authoring adrs easily.